data-pipeline/bcreg/bcregistries.py

```diff
@@ -741,6 +742,7 @@ def get_max_date_before_now(self):
         try:
             # create a cursor
             cur = self.conn.cursor()
+            cur.execute("""SET TIME ZONE """ + BC_REGISTRIES_TIMEZONE)
```

Comment:
I'd recommend switching to using the cursor inside a `with` block so `.close()` is calls automatically and not explicitly.  That way the session and the time-zone setting are finite even if there is an error.

Reply:
Good point